charts/gardener/provider-local/templates/deployment.yaml

```diff
@@ -32,6 +32,7 @@ spec:
         networking.gardener.cloud/to-runtime-apiserver: allowed
         networking.gardener.cloud/to-dns: allowed
         networking.gardener.cloud/to-kind-network: allowed
+        networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443: allowed
```

Comment:
We should add notes to the commit messages, when the respective commits can be dropped again.

Reply:
I've added a remark in the commit message of the first two commits.